clang/test/Sema/attr-nolock-wip.cpp

```diff
@@ -0,0 +1,26 @@
+// RUN: %clang_cc1 -fsyntax-only -fblocks -verify %s
+// R UN: %clang_cc1 -fsyntax-only -fblocks -verify -x c -std=c23 %s
+
+#if !__has_attribute(clang_nolock)
+#error "the 'nolock' attribute is not available"
+#endif
+
+void unannotated(void);
+void nolock(void) [[clang::nolock]];
```

Comment:
`(void)` instead of `()` is unnecessary if you’re only testing C++ and C23.

Reply:
Wow, I never thought I'd see this day!

Reply:
It took us 50 years, but we got there eventually haha.